system-int-test/test/cmr/system_int_test/search/collection_doi_search_test.clj

```diff
@@ -50,4 +58,15 @@
        [coll1 coll2] ["Doi*"] {:pattern true}
 
        "search for collections with both doi1 and doi2 returns nothing"
-       [] ["Doi1" "doI2"] {:and true}))))
+       [] ["Doi1" "doI2"] {:and true}))
+    (testing "search doi with json query"
+      (let [not-provided-results (search/find-refs-with-json-query
```

Comment:
Change this to use `are3`. Add more tests similar to what doi search param above.